servicetalk-http-netty/src/test/java/io/servicetalk/http/netty/TracingHttpServiceFilterTest.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright © 2021 Apple Inc. and the ServiceTalk project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.servicetalk.http.netty;
+
+import io.servicetalk.opentracing.http.TracingHttpServiceFilter;
+import io.servicetalk.opentracing.inmemory.DefaultInMemoryTracer;
+
+import org.junit.Test;
+
+import static io.servicetalk.http.netty.AsyncContextHttpFilterVerifier.verifyServerFilterAsyncContextVisibility;
+import static io.servicetalk.opentracing.asynccontext.AsyncContextInMemoryScopeManager.SCOPE_MANAGER;
+
+public class TracingHttpServiceFilterTest {
```

Comment:
Oups, found one thing:
```
java: Annotation processing is not supported for module cycles. Please ensure that all modules from cycle [servicetalk-opentracing-http,servicetalk-http-netty] are excluded from annotation processing
```
when tried to run any test from `http-netty`

Reply:
We should have a way to verify those during the build. I am running them with Idea (not Gradle) and they work and that's a false promise :(

Reply:
Did you do `./gradlew idea` before trying to run intellij idea test?

Reply:
I meant I am running them with the IntelliJ test runner so it doesn't complain.

Reply:
It really depends on how your gradle project in imported in Intellij IDEA. I do not use built-in Intellij feature for gradle import. Instead, I run `./gradlew idea`. It means that if I change anything in gradle files, I have to re-run this command manually in console to let Intellij see the changes.